SolarTherm/Models/CSP/CRS/Receivers/PBS_Receiver.mo

```diff
@@ -0,0 +1,126 @@
+within SolarTherm.Models.CSP.CRS.Receivers;
```

Comment:
what is "PBS receiver"? please add some comments/docs for context.

Reply:
assume it's packed bed storage, please clarify why a specific receiver model is needed for this.